app/packages/map/src/Map.tsx

```diff
@@ -159,7 +159,14 @@ const Plot: React.FC<{}> = () => {
   }, [data]);
 
   if (!settings.mapboxAccessToken) {
-    return <foc.Loading>No Mapbox token provided</foc.Loading>;
+    return (
+      <foc.Loading>
```

Comment:
Looks good! But an `ExternalLink` will need to be used instead of a regular `a` element because FiftyOne is also distributed as an electron app. The component is available in `@fiftyone/components`.